components/collection/drop/PaidGenerative.vue

```diff
@@ -26,63 +26,39 @@
   <CollectionDropModalPaidMint
     v-model="isMintModalActive"
     :action="action"
-    :to-mint-nft="toMintNft"
-    :is-allocating-raffle="isAllocatingRaffle"
+    :to-mint-nfts="toMintNFTs"
+    :amount-to-mint="amountToMint"
+    :minting-session="mintingSession"
+    :ready-to-mint="canMint"
```

Comment:
some of these are coming from `useDropMintRefs`

this is an anti patten, and is bringing us again in the direction of many props

 
 why not use the composble/s inside `PaidMint`?